src/rest_service/result/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+import uuid
+
+# Create your models here.
+
+class Result(models.Model):
+    id = models.UUIDField(
+        _("Id"),
+        primary_key=True,
+        db_column="result_id",
+        db_index=True,
+        auto_created=True,
+        unique=True,
+        editable=False,
+        default=uuid.uuid4
+    )
+
+    # In team_id ForeignKey cant write 'team.Team.id'
+    # is 'team.Team' ok?
+    # This goes for scenario_id too..
```

Comment:
@hiddenSm 
yes that attr is to define table not column of foreignkey
to do that you can specify another field which i don't remember 
however if you don't specify that it will set the foreignkey to the primary